1209/C1AA/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Computadora Industrial Abierta Argentina
+owner: Proyecto CIAA
```

Comment:
'Owner' needs to be the name of the directory in which the org file is held - in this case, 'ciaa'.